rust/src/lib/ifaces/linux_bridge.rs

```diff
@@ -167,7 +167,11 @@ impl LinuxBridgeInterface {
             .as_ref()
             .and_then(|br_conf| br_conf.port.as_ref())
             .map(|port_confs| {
-                port_confs.iter().any(|port_conf| port_conf.vlan.is_some())
+                port_confs.iter().any(|port_conf| {
+                    port_conf.vlan.is_some()
+                        && *port_conf.vlan.as_ref().unwrap()
```

Comment:
unwrap is not allowed in library

Reply:
Fixed, but how can be the test `test_disable_vlan_filtering_on_bridge` being passing without this fix ?

Reply:
Well the tests was not testing if vlan-filtering was activated, it's missing something like 

```bash
[ellorent@fedora rust]$ nmcli c show br1 |grep vlan 
bridge.vlan-filtering:                  yes
```

Reply:
Added a check for the vlan-filtering flag